File: upeu/Persona.py
class Persona:
    nombre:str
    genero:str
    ocupacion:str

    def __init__(self):
        pass

    def __init__(self, nombre):
        self.nombre=nombre

    def __init__(self, nombre, genero, ocupacion):
        self.nombre=nombre
        self.genero=genero
        self.ocupacion=ocupacion

    def obtenerNombre(self):
        return self.nombre
    def obtenerGenero(self):
        return self.genero
    def obtenerOcupacion(self):
        return self.ocupacion

    def modificarNombre(self, nombre):
        self.nombre=nombre
    def modificarGenero(self, genero):
        self.genero=genero
    def modificarOcupacion(self,ocupacion):
        self.ocupacion=ocupacion


myObj = Persona()
myObj.nombre="David"
print(myObj.nombre)
myObj.modificarNombre("Raul")
print(myObj.nombre)
print(myObj.obtenerNombre())

File: upeu/Persona.java
package upeu;
/**
 * Persona
 */
public class Persona {
    String nombre;
    char genero;
    String ocupacion;

    Persona(){        
    }

    Persona(String nombre){
        this.nombre=nombre;
    }
    Persona(String nombre, char genero, String ocupacion){
        this.nombre=nombre;
        this.genero=genero;
        this.ocupacion=ocupacion;
    }

    public String obtenerNombre() {
        return nombre;
    }

    public char obtenerGenero() {
        return genero;
    }

    public String obtenerOcupacion() {
        return ocupacion;
    }

    public void modificarNombre(String nombre) {
        this.nombre=nombre;
    }

    public void modificarGenero(char genero) {
        this.genero=genero;
    }

    public void modificarOcupacion(String ocupacion) {
        this.ocupacion=ocupacion;
    }        
}

class Principal{
    public static void main(String[] args) {
        Persona myObj=new Persona("Dario",'M', "Ing. Civil");
        System.out.println(myObj.obtenerNombre());
        System.out.println(myObj.obtenerGenero());
        System.out.println(myObj.obtenerOcupacion());
        /* .out.println(myObj.nombre);
        myObj.modificarNombre("Raul");
        System.out.println(myObj.nombre);
        System.out.println(myObj.obtenerNombre());*/
    }
}

